pkg/webhook/mutation/pod/oneagent/mutator.go

```diff
@@ -66,8 +66,10 @@ func (mut *Mutator) Mutate(ctx context.Context, request *dtwebhook.MutationReque
 
 	log.Info("injecting OneAgent into pod", "podName", request.PodName())
 
-	if err := mut.ensureInitSecret(request); err != nil {
-		return err
+	if !request.DynaKube.FeatureDownloadViaJob() || request.DynaKube.OneAgent().IsCSIAvailable() {
```

Comment:
no changes are necessary here, we will not use the `[oneagent/mutator.go](https://github.com/Dynatrace/dynatrace-operator/pull/4515/files#diff-4f19cd6303965594c593fbf1ac8b2b13cc8691d00a6f181ee2d5522cca948104)` when this feature is turned on

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4515/commits/3b757cd16ce2bf8c3e839370a7c683e66b2d5dce